src/core/utils/remote-feature-config.ts

```diff
@@ -0,0 +1,41 @@
+import firebase from 'react-native-firebase';
+import DeviceInfo from 'react-native-device-info';
+
+export enum REMOTE_FEATURE {
+    NEAR = 'feature_near'
+}
+
+export const getRemoteConfigFeatures = async () => {
+    // TODO: decide if 1 hour should be the optimal fetch duration
+    // set fetch cache duration to 1 hour for live environment
+    let duration = 3600;
+    if (__DEV__) {
+        firebase.config().enableDeveloperMode();
+        duration = 0;
+    }
+
+    await firebase.config().fetch(duration);
+    await firebase.config().activateFetched();
+    const objects = await firebase.config().getValues([REMOTE_FEATURE.NEAR]);
+
+    const data = {};
+    // Retrieve values
+    Object.keys(objects).forEach(key => {
+        data[key] = objects[key].val();
+    });
+
+    return data;
+};
+
+export const isFeatureActive = async (feature: REMOTE_FEATURE): Promise<boolean> => {
```

Comment:
this shoudn't be async, we could call `getRemoteConfigFeatures` on app bootstrap and cache the data, after that `isFeatureActive` could use cached data. It's easier and more user friendly this way.

Reply:
by doing this function sync, it will be easier to use it in components.